### PR TITLE
New Doc: Wordpress S3

### LINKED ISSUE #3423: New Doc Proposal — S3 on WordPress
Priority (Low‚ Medium‚ High): Low

## Title
AWS S3 Setup for WordPress

## Description
Document how to configure WordPress to offload file storage to AWS S3. Similar to our [doc for Drupal]( https://pantheon.io/docs/drupal-s3/).

Some of the Drupal doc content would apply for WP as well ("Before you begin" and "Configure S3 within the AWS Console," at least).

## Outline
* Before you begin (could be shared with Drupal's doc)
* Configure S3 within the AWS Console (could be shared with Drupal's doc)
* Integrate S3 with WordPress
* Plugin configuration
* Synchronizing the S3 Bucket and WordPress Files

## Expected Audience
WordPress developers/site maintainers

## Path
`source/docs/wordpress-s3`

### PR DESCRIPTION
**Pull request by sarahg**
Closes #3423

## Effect
PR includes the following changes:
- Adds new doc about using S3 to offload WordPress file storage

## Remaining Work
- [x] Finish writing content
- [x] Test process

## Docs Team Work
- [x] Technical Review
- [x] Copy Review

### CODE REVIEW

**Comment by rachelwhitton** [comment]
@sarahg noting this is still in progress, just wanted to say that it looks really great so far!! Your attention to detail is 💯 

Is there anything I can do to help finish this one out?

**Comment by sarahg** [comment]
@rachelwhitton I got a little side-tracked here by the fact that the S3 Uploads plugin I'm writing about has a conflict with our Solr Power plugin (similar to this: https://github.com/humanmade/S3-Uploads/issues/80) and installing them both as-is will bork a site. I've been working on a fix/recommendation for that but should probably focus back on the issue at hand here, the S3 part. Will try to move this forward this week!

**Comment by sarahg** [comment]
@alexfornuto this one is (finally) ready for review! 🙀

File: source/_docs/wordpress-s3.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <p markdown="1">**Do not** add the plugin as a Git submodule. Git submodules are not supported on the platform ([more info](https://pantheon.io/docs/git-faq/#does-pantheon-support-git-submodules)).</p>
+    </div>
+
+2. Rename the extracted folder to remove the version number. For example:
+
+    ```bash
+    mv S3-Uploads-2.0.0/ S3-Uploads
+    ```
+
+3. Create and / or copy your S3 upload key and secret from the "My security credentials" section of your AWS account to a text editor on your local computer.
```
Comment (alexfornuto) [inline source/_docs/wordpress-s3.md]: @sarahg I think this is where I'm getting stuck. I cannot find any "Upload Key" information on AWS. 

While on the subject, it seems like AWS is strongly pushing me to have an "IAM" user to associate with this setup. Thoughts?

Comment (sarahg) [reply source/_docs/wordpress-s3.md]: @alexfornuto Good catch! The plugin refers to them a bit differently than the AWS UI does. Adjusted the label in 4acec05, can you see if that makes it clearer?

As for the IAM user — that gets created via a WP-CLI command a little further down the line (`terminus wp <site>.<env> -- s3-uploads create-iam-user`). Might be worth noting that at this point, too, though?

**Pull request merged**